### PR TITLE
WIP: fix: update to latest angular commit conventions

### PR DESCRIPTION
**Pull request by marionebl**
* https://github.com/angular/angular/commit/dff6ee32725197bdb81f3f63c5bd9805f2ed22bb#diff-6a3371457528722a734f3c51d9238c13L204
* https://github.com/marionebl/commitlint/commit/d4bdcf66f3355dd4328b52a1ed986f79acb079b1
* https://github.com/marionebl/conventional-commit-types/commit/21740af2547c535ccadf72c47c8cd151fa2972ea

### CODE REVIEW

File: packages/conventional-changelog-cli/README.md [hunk 1]
```diff
@@ -82,7 +82,7 @@ By adding a `.npmrc` you could also automate your commit message and set your ta
 
 ```
 tag-version-prefix=""
-message="chore(release): %s :tada:"
+message="build(release): %s :tada:"
```
Comment (stevemao) [inline packages/conventional-changelog-cli/README.md]: I think release should be `chore`

Comment (stevemao) [reply packages/conventional-changelog-cli/README.md]: You can probably change this to `chore(release): publish v1.0.0`

**Comment by marionebl** [comment]
Angular removed the chore type completely, that was the motivation for this change.

**Comment by marionebl** [comment]
I am not happy with the `build` type for releases, opened an issue with Angular about this: https://github.com/angular/angular/issues/20537

Keeping this WIP until the issue is resolved.

**Comment by marionebl** [comment]
Closing in favor of the fork described in https://github.com/conventional-changelog/standard-version/pull/220

**Comment by hbetts** [comment]
@marionebl isn't this change still needed in `conventional-changelog-angular` to keep the package up-to-date with Angular's conventions?

Or is there an effort to deprecate the `conventional-changelog-angular` package in favor of another preset package for use as a default?

**Pull request closed without merging**